### ISSUE TITLE
Can't find clang14 package

### ISSUE THREAD

**Issue by spvkgn** [ivanvc/abuild-releaser-action#2]
[clang14](https://pkgs.alpinelinux.org/package/v3.19/main/x86_64/clang14) in main repo - no errors when install it manually in alpine:latest docker image.

GH action failed: https://github.com/spvkgn/alpine-packages/actions/runs/9170634757
```
>>> uasm: Analyzing dependencies...
2024-05-21T07:28:23.6433415Z ERROR: unable to select packages:
2024-05-21T07:28:23.6436101Z   .makedepends-uasm-20240521.072823:
2024-05-21T07:28:23.6437243Z     masked in: cache
2024-05-21T07:28:23.6439026Z     satisfies: world[.makedepends-uasm=20240521.072823]
2024-05-21T07:28:23.6440470Z   clang14 (no such package):
2024-05-21T07:28:23.6441431Z     required by: .makedepends-uasm-20240521.072823[clang14]
2024-05-21T07:28:23.6538285Z >>> ERROR: uasm: builddeps failed
```

**Comment by ivanvc** [comment]
Hi @spvkgn, `clang14` is not available anymore in edge. Can you use `clang15` instead? Supporting pining to a specific Alpine Linux version would not be an easy fix. So, I suggest using a version that is in the edge repository.

**Comment by spvkgn** [comment]
Hello @ivanvc I've tried just to add support for Alpine branch as input and this did the trick https://github.com/spvkgn/abuild-releaser-action/commit/617ea4c025bb014f838e249ba85865e75e0bec90

**Comment by ivanvc** [comment]
> Hello @ivanvc I've tried just to add support for Alpine branch as input and this did the trick [spvkgn@617ea4c](https://github.com/spvkgn/abuild-releaser-action/commit/617ea4c025bb014f838e249ba85865e75e0bec90)

Nice! Thank you. Could you open a PR, so I can get it merged? Thanks again @spvkgn!

**Comment by ivanvc** [comment]
Thanks for your PR @spvkgn! I published your fix as [v1.2.0](https://github.com/ivanvc/abuild-releaser-action/releases/tag/v1.2.0)
